src/oci-umount.c

```diff
@@ -393,8 +393,6 @@ static int unmount(const char *id, char *umount_path, bool submounts_only, const
 		ret = umount2(mnt_table[i].destination, MNT_DETACH);
 		if (!ret)
 			pr_pinfo("%s: Unmounted submount: [%s]\n", id, mnt_table[i].destination);
-		else
-			pr_perror("%s: Failed to unmount submount: [%s]. Skipping.\n", id, mnt_table[i].destination);
```

Comment:
Why this is expected failure?

Reply:
This is the commit I am seeing in latest commits. So why are we removing this error message?

Reply:
This message is coming up in the Hide Outdated, I have no idea why

Reply:
Ok when I reformated the patches, one removed this code and one put it back,  They should be clean now.